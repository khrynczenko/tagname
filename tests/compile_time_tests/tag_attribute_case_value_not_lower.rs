use tagname::TagName;

#[derive(TagName)]
enum Cases {
    #[tag(case = "something")]
    Lower,
    Upper
}

fn main() {
}
