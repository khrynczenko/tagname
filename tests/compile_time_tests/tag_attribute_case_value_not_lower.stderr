error: `case` value must be either "upper" or "lower"
 --> tests/compile_time_tests/tag_attribute_case_value_not_lower.rs:5:18
  |
5 |     #[tag(case = "something")]
  |                  ^^^^^^^^^^^
